tests/test_cli_compile.py

```diff
@@ -1656,3 +1656,29 @@ def test_triple_equal_pinned_dependency_is_used(
     assert out.exit_code == 0, out
     for line in out_expected_content:
         assert line in out.stderr
+
+
+@pytest.mark.network
+def test_flit(runner):
```

Comment:
The body of both tests looks similar. Seems like we could combine `test_setup_cfg` and `test_flit` via `pytest.mark.parametrize`.

Reply:
Great catch! I parametrized tests, dropped auxiliary files, added a test case for poetry.